test/test-applications/regression/StackExchange.Redis.AssemblyConflict.SdkProject/Program.cs

```diff
@@ -17,16 +17,16 @@ public static async Task Main()
             }
             finally
             {
-                var instrumentationType = Type.GetType("Datadog.Trace.ClrProfiler.Instrumentation, Datadog.Trace.ClrProfiler.Managed");
+                var instrumentationType = Type.GetType("Datadog.Trace.ClrProfiler.Instrumentation, OpenTelemetry.Instrumentation.ClrProfiler.Managed");
```

Comment:
I think that this change is wrong. The name of the assembly has changed. Not the type's FullName.

Reply:
Not longer valid (per other comments)